t/full-stack.t

```diff
@@ -169,6 +169,7 @@ subtest 'wait until developer console becomes available' => sub {
     OpenQA::Test::FullstackUtils::wait_for_developer_console_available($driver);
 };
 
+sleep 8;
```

Comment:
which command fails without the sleep, the line L185 waiting for the log message in developer console because the command in L183 is lost? Could we instead of sleep send the command and wait for the message multiple times in case the first is lost?

Reply:
Yes, the tests in L185 and L192 are failing.

The command is sent with `send_json()`, but nothing is reading it.
You can look at the test artifact from circleci here https://circleci.com/workflow-run/3288d3c6-5ef5-455c-9e87-edaf61751cb4, I dump the content of autoinst-log.txt after the test.

Not sure yet how I would do the "send the command and wait" multiple times.

Reply:
I think if the backend is losing commands, we should try to fix it there.
Or alternatively find a way to add information to the status info if the backend is ready to take commands.

Reply:
If I add the HTTP status call in the worker again, I can see that the pause command is sent later (the line in the test is executed later), and it passes without the sleep.
So the status call has some effect that keeps the webui busy/waiting.

Reply:
That `sleep` is seemingly outside of a specific test so I feel like we should discuss that to find an alternative to it... there's the following `wait_fr_result_panel` which already has a timeout/ re-try loop, and there's `pause at certain test` which locates elements and sends keys initially - is this where we need an additional delay?

Reply:
This pull request was what I had when I found out what was the problem, and I wanted to push something to fix the current failing test as soon as possible.
I know it shouldn't be our permanent solution to this.

Reply:
Fair enough. In that case I'd appreciate a `FIXME` comment in the code, though. This PR isn't very clear in that regard :smile:

Reply:
please don't add FIXME in codes. We all know that this means we never look again ;) We have tickets for that :)

Reply:
The description doesn't mention a ticket either...